Add database shape

coldacid opened the pull request:
Based on @thorsil's PR #2 but based on the current styling and structure. Merging this should obsolete #2.

For additional shapes, I wouldn't have any suggestions that would work with PlantUML at this time. I'd love to have a folder shape for file system elements, but the actual `folder` shape in PlantUML sticks the label in the label space of the folder icon instead of in the main box. Unless PlantUML starts providing the ability to add custom shapes instead of just sprites, adding the database shape is about all we can do.

RicardoNiepel commented:
Can you please resolve the conflicts and also remove the shadow?
I will then merge it.

coldacid commented:
I've rebased the branch to eliminate the merge conflict, and turned off shadows for the database elements. I also added the database macros to the lists in the readme file since I noticed that I had forgotten to add them there. Should be good to merge now!
